spec/features/contact_us_when_not_signed_in_spec.rb

```diff
@@ -119,6 +119,8 @@
   end
 
   context "with incorrect email formats" do
+    let(:zendesk_error) { ZendeskAPI::Error::RecordInvalid }
```

Comment:
Is this doing anything? Below the class is referenced directly.

Reply:
uh good point. I'll remove